guides/v2.1/contributor-guide/templates/basic_template.md

```diff
@@ -12,6 +12,23 @@ The purpose of this page is to provide you with a pre-formatted template and use
 
 You can start off by editing the local version of this file using markdown language (and {% glossarytooltip a2aff425-07dd-4bd6-9671-29b7edefa871 %}HTML{% endglossarytooltip %} where needed). Then, create a Pull Request to have your contribution reviewed by the DevDocs team.
 
+## Metadata parameters
+
+Add the following {% glossarytooltip 3f0f2ef1-ad38-41c6-bd1e-390daaa71d76 %}metadata{% endglossarytooltip %} parameters at the top of your page. We use this data when generating the DevDocs for the following:
```

Comment:
I don't see _metadata_ in the [Glossary](https://docs.magento.com/m2/ce/user_guide/backmatter/glossary.html)
Remove `glossarytooltip`

Reply:
Removed. Also removed it from the tutorial first page file (new in changed files for this PR).